core/tasks.py

```diff
@@ -238,3 +243,31 @@ def _overall_score_and_result_type(datafile):
     result_type = ScoreResult.ResultTypes.SIMPLE
     overall_score = float(datafile.readline())
     return overall_score, result_type
+
+
+def _get_mimetype(file_path):
+    mimetype = magic.from_file(file_path, mime=True)
+    # check if mimetype has something useful
+    # empty file returns inode/x-empty
+    if mimetype == 'inode/x-empty':
+        # if no mime type, it can be null(None)
+        return None
+    # Unsure about how to get file and content mime type seperately
+    mime_type = mimetype.split('/')
```

Comment:
The minetype is expected to have a slash in it.  We don't want to split it.

Reply:
I thought the slash is to separate the file mime type and content mime type. So the whole thing together is a mime type for the file?

Reply:
Yes, the whole thing is the mime type.  Pedantically, it is a type and subtype, but they are almost always used as a single value.